htdocs/core/lib/resource.lib.php

```diff
@@ -146,3 +146,53 @@ function resource_admin_prepare_head()
 
 	return $head;
 }
+
+/**
+ * Returns the resources marked "busy" and assigned to an event during the specified time frame.
+ * You can specify an array of resources and the function will return the overlap of the two.
+ * @param $dateStart string
+ * @param $dateEnd string
+ * @param $resource_ids array
+ * @return array|bool
+ */
+function get_busy_resource_during(string $dateStart, string $dateEnd, $resource_ids = array())
```

Comment:
Introduced new function must nowuse the camel case syntax. Can you rename it into this name
`getBusyResourcesInPeriod`